swagger.yml

```diff
@@ -3548,31 +3609,51 @@ paths:
             application/json:
               schema:
                 type: object
-                required:
-                  - Features
-                  - Technologies
-                  - Alerts
                 properties:
                   Features:
-                    $ref: '#/components/schemas/Features' 
-                  Alerts: 
-                    $ref: '#/components/schemas/AggregatedCodeSeverityAlerts'
-                  Technologies:
+                    $ref: '#/components/schemas/Features'
+                  technologies:
                     type: object
-                    required:
-                      - Technologies
-                      - Total
+                    description: Available technologies
```

Comment:
use the word 'supported' instead of the word 'available'.

Reply:
ok